projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/google-sheets-card.js

```diff
@@ -5,7 +5,8 @@ import GoogleSheetsIcon from '../../../../icons/google-sheets';
 import IntegrationCard from './integration-card';
 
 const FORM_RESPONSES_URL =
-	window?.jpFormsBlocks?.defaults?.formsResponsesUrl || '/wp-admin/admin.php?page=jetpack-forms';
+	window?.jpFormsBlocks?.defaults?.formsResponsesUrl ||
+	'/wp-admin/admin.php?page=jetpack-forms-admin';
```

Comment:
Do these URLs work for sites in folders? It would be good to check that. :)

Reply:
It would be good if this `window?.jpFormsBlocks?.defaults?.formsResponsesUrl` was always present. 
how so that we can we don't have to do this `	'/wp-admin/admin.php?page=jetpack-forms-admin';`

Reply:
Should this URL also include the url hash `#responses` ?

Reply:
The value is usually there, this is a fallback URL just in case anything fails. The PR is just updating what was there before to the new menu slug.